backend/src/main/java/com/crm/api/v1/CustomerController.java

```diff
@@ -50,4 +51,12 @@
                         .toUri())
                 .body(customerResponse);
     }
+
+    @PutMapping("/{id}")
+    @ApiOperation(value = "Updates existing customer", notes = "Add body \"CustomerRequest\" and path variable \"id\""
+            + " of existing customer to proceed the update")
+    public ResponseEntity<CustomerResponse> updateCustomer(@Valid @RequestBody final CustomerRequest customerRequest,
+                                                           @PathVariable final Long id) {
+        return ResponseEntity.ok(customerService.updateCustomer(customerRequest, id));
```

Comment:
```suggestion
       CustomerResponse updatedCustomer = customerService.updateCustomer(customerRequest, id);
       
        return ResponseEntity.ok(updatedCustomer);
```

Ja bym tutaj najpierw zapisał to do zmiennej, żeby lepiej przedstawić, czym jest ciało odpowiedzi

Reply:
Teraz zrobiłeś to tak:

```java
CustomerResponse customerResponse = customerService.updateCustomer(customerRequest, id);
return ResponseEntity.ok(customerResponse);
```

Moim zdaniem moja nazwa `updatedCustomer` jest lepsza, ponieważ informuje czytelnika o tym czym jest ciało odpowiedzi (oraz co jest zwracane przez `CustomerService#updateCustomer()`).